### PR TITLE
chore(deps): update viz to v0.10

### PR DESCRIPTION
**Pull request by fundon**

### CODE REVIEW

File: rust/viz/config.yaml [hunk 1]
```diff
@@ -1,3 +1,3 @@
 framework:
   website: viz.rs
-  version: 0.9
+  version: 0.10
```
Comment (waghanza) [inline rust/viz/config.yaml:3]:
```suggestion
  version: '0.10'
```


Otherwise 0.1 will be printed

**Review by waghanza — CHANGES_REQUESTED** [review]

File: rust/viz/config.yaml (hunk 1 above)
Comment (fundon) [reply rust/viz/config.yaml]: thanks

File: rust/viz/config.yaml [hunk 2]
```diff
@@ -1,3 +1,3 @@
 framework:
   website: viz.rs
-  version: 0.9
+  version: "0.10.0"
```
Comment (waghanza) [inline rust/viz/config.yaml:3]:
```suggestion
  version: "0.10"
```

**Review by waghanza — CHANGES_REQUESTED** [review]

File: rust/viz/config.yaml (hunk 2 above)
Comment (fundon) [reply rust/viz/config.yaml]: does not work

**Review by waghanza — APPROVED** [review]

**Pull request merged**